src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.ImportExport.cs

```diff
@@ -0,0 +1,558 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.Cryptography;
+using Internal.NativeCrypto;
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    using BCRYPT_ECCFULLKEY_BLOB = Interop.BCrypt.BCRYPT_ECCFULLKEY_BLOB;
+    using BCRYPT_ECCKEY_BLOB = Interop.BCrypt.BCRYPT_ECCKEY_BLOB;
+    using ErrorCode = Interop.NCrypt.ErrorCode;
+    using KeyBlobMagicNumber = Interop.BCrypt.KeyBlobMagicNumber;
+
+    public sealed partial class ECDsaCng : ECDsa
+    {
+        private static CngKey ImportKeyBlob(byte[] ecBlob, ECNamedCurve curve, bool includePrivate)
```

Comment:
Correct. The normal (not the explicit) key blob apis were already there and they didn't extend the buffer to hold the key. I found (partially through trial and error as there is no doc afaik) it has to be set via NCryptImportKey as one of the extra 'buffers' you can pass in (see CngKey.Import.cs)